src/main/java/me/moodcat/api/models/SongModel.java

```diff
@@ -66,6 +66,12 @@
      */
     private String artworkUrl;
 
+    @Deprecated
+    private double valence;
+
+    @Deprecated
+    private double arousal;
```

Comment:
We might want to leave this in for a while for debugging? So we can see if the sorting works on larger datasets?